src/utils/checkAnswer.js

```diff
@@ -0,0 +1,10 @@
+import { quizData } from '../data.js';
+
+export const checkAnswer = (questionNum) => {
+  const answer = localStorage.getItem(questionNum);
```

Comment:
Replaces `questionNum` with `questionId`

Reply:
questionNum is question id that is passed form other component, So i think that is more appropriate.